docs/IE_DG/Extensibility_DG/AddingNGraphOps.md

```diff
@@ -16,6 +16,8 @@ To add your custom nGraph operation, create a new class that extends `ngraph::Op
 
 5. Override the `visit_attributes` method, which allows serialization and deserialization of attributes. An `AttributeVisitor` is passed to the method, and the implementation is expected to walk over all the attributes in the op using the type-aware `on_attribute` helper. Helpers are already implemented for standard C++ types like `int64_t`, `float`, `bool`, `vector` and for existing nGraph defined types.
 
+6. Override the `evaluate` method, this method allows to apply constant folding, if the custom operation exists on the constant branch. This is an optional method.
```

Comment:
--> ```Override `evaluate`, which is an optional method that enables the application of constant folding if there is a custom operation on the constant branch.```

Please check if the meaning didn't change

Reply:
Fixed.